pkg/apis/dashboard/v0alpha1/types.go

```diff
@@ -0,0 +1,124 @@
+package v0alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	runtime "k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/grafana/grafana/pkg/apis"
+)
+
+const (
+	GROUP      = "dashboard.grafana.app"
+	VERSION    = "v0alpha1"
+	APIVERSION = GROUP + "/" + VERSION
+)
+
+var DashboardResourceInfo = apis.NewResourceInfo(GROUP, VERSION,
+	"dashboards", "dashboard", "Dashboard",
+	func() runtime.Object { return &Dashboard{} },
+	func() runtime.Object { return &DashboardList{} },
+)
+
+var DashboardSummaryResourceInfo = apis.NewResourceInfo(GROUP, VERSION,
+	"summary", "summary", "DashboardSummary",
```

Comment:
is this too short of a short name? it's probably fine given dashboard's position in the grafana ecosystem, but I feel like for anything else we'd name it x-summary or x-list-item

Reply:
i think we can register a "short name" "dashboard-summary" so that it works easily with kubectl, but adding dashboard the singular+plural name will make the URLs pretty silly.